v2/thirdparty/custom-ui/multi-tenancy.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: multi-tenancy
+title: "Multi-tenancy Implementation"
+hide_title: true
+---
+# Multi-tenancy Implementation
+
+## What is multi-tenancy? and, Possible flows
+
+diagram
+
+- All tenants on Single Domain
+- Each tenant on a different sub-domain
+  - Handling callbacks through common domain
+
+## Config management using Core
+
+- CRUD operations
+- Config structure
+- Note on `clientType`
+
+## Frontend
+
+- getConfiguredProviders API
+
+## Backend
+
+- Automatic creation of providers for built-in
+- Using Custom providers for multi-tenancy
+
+## Session handling
+
+- Session claims to assert tenant
```

Comment:
Have a "See also" section in this page which links to:
- Setup common domain for all tenants
- Setup sub domain per tenant
- Custom multi tenant providers
- (And all other sections that are there in https://supertokens.com/docs/thirdpartyemailpassword/custom-ui/thirdparty-login#see-also section